ts/client/scripts/archive/devnet-user-2.ts

```diff
@@ -71,9 +72,14 @@ async function main() {
 
   // create + fetch account
   console.log(`Creating mangoaccount...`);
-  const mangoAccount = await client.getOrCreateMangoAccount(group);
+  const mangoAccount = (await client.getMangoAccountForOwner(
```

Comment:
this is no longer creating accounts, right? but the whole script is in archive/ so...